kbase/narrative
# DATAUP-288 left column file type panel

# Description of PR purpose/changes

This includes a component that's, generically, a toggleable category panel. It's used here as the file type selector.

Right now the component itself works as described in the ticket, but the bulk import cell doesn't do too much toggling based on the selection. It's also hard-coded to have FASTQ and SRA file types only, right now (that's the next step to do).

Another issue I'm having is that we're on FontAwesome 4.7.0, which doesn't have the DNA icon that goes up in the header. I think upgrading to the free version of 5 is a great idea, but might also be kinda onerous as it's not backward compatible. I'm not sure what to do about that yet, but I'll ask design. I just wanted to get this PR set up first.

It should look roughly like this (ignore the rest of the incomplete layout...)
![Screen Shot 2020-11-19 at 12 03 56 PM](https://user-images.githubusercontent.com/2152243/99718210-6a620300-2a5f-11eb-9177-899893492e6f.png)


# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X

# Testing Instructions
* Details for how to test the PR: 
open a bulk import cell, and it should appear

# Dev Checklist:
